ankisyncd/users.py

```diff
@@ -65,13 +78,14 @@ def user_exists(self, username):
         return username in users
 
     def del_user(self, username):
+        # Warning, this doesn't remove the user directory or clean it
         if not self.auth_db_exists():
             raise ValueError("Auth DB {} doesn't exist".format(self.auth_db_path))
 
-        conn = sqlite.connect(self.auth_db_path)
+        conn = self._conn()
         cursor = conn.cursor()
         logging.info("Removing user '{}' from auth db".format(username))
-        cursor.execute("DELETE FROM auth WHERE user=?", (username,))
+        cursor.execute(self.fs("DELETE FROM auth WHERE username=?"), (username,))
```

Comment:
This is backwards incompatible for no good reason. I do agree `username` is a better name for this field, though.

Reply:
Actually `user` is a reserved keyword in a number of (most?) SQL dialects, meaning it can't be a column name. But I agree just introducing a backwards-incompatible change with no mitigation is wrong. I wanted to avoid dealing with that via dynamic replaces in `def fs` but can do it that way. If you have a better idea, I'm all ears though!

Reply:
Okay, that is a good reason. I'd prefer the column names to stay the same for all backends, though (unless they're overriding all SQL-calling methods anyway). Do you think migrating existing sqlite3 databases to use `username` would be too bothersome?

Reply:
I have added a script, what do you think?